src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -80,7 +81,10 @@ function NewContactMethodPage(props) {
             Navigation.navigate(ROUTES.SETTINGS_CONTACT_METHODS);
             return;
         }
-        User.addNewContactMethodAndNavigate(login, password);
+        const phoneLogin = LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(login));
+        const parsedPhoneNumber = parsePhoneNumber(phoneLogin);
```

Comment:
Can we move that to be before the "If this login already exists, just go back." block.